### PR TITLE
Send altered confirmation email if not being sent.

### LINKED ISSUE #1210: Users still receiving confirmation emails when sending reports to councils that no longer accept them
It seems that users still receive a confirmation email even though we let them know we can't send report to council.

Perhaps wording should change to: 

Hello [user],

Please click on the link below to confirm that you want your report to appear on FixMyStreet, despite not being sent to the council:

https://www.fixmystreet.com/P/DjL2SGBQjxKXBfn7pA

If your email program does not let you click on this link, copy and
paste it into your web browser and press return.

Your problem had the title:

[ ...]

And details:

[.....]

If you no longer wish your report to be displayed on the site, please take no further
action.

Thank you for supporting FixMyStreet.

All the best,

The FixMyStreet team

https://www.FixMyStreet.com Twitter: https://twitter.com/FixMyStreet

Facebook: http://www.facebook.com/fixmystreet

Problems? questions?

Visit https://www.fixmystreet.com/faq

This email was sent automatically, from an unmonitored email account, so please do not reply to it.

### PR DESCRIPTION
**Pull request by dracos**
Fixes #1210.

### CODE REVIEW

**Comment by dracos** [comment]
This needs a test!

File: t/app/controller/report_new.t [hunk 1]
```diff
@@ -1494,10 +1495,17 @@ subtest "unresponsive body handling works" => sub {
...
+        my $user = FixMyStreet::App->model('DB::User')->find( { email => $test_email } );
+        ok $user, "test user does exist";
+
         my $report = $user->problems->first;
         ok $report, "Found the report";
         is $report->bodies_str, undef, "Report not going anywhere";
 
+        my $email = $mech->get_email;
+        ok $email, "got an email";
+        like $email->body, qr/despite not being sent/i, "correct email sent";
```
Comment (davea) [inline t/app/controller/report_new.t]: Worth including a check that this *doesn't* appear in one of the other tests?

**Comment by davea** [comment]
:+1:

File: t/app/controller/report_new.t (hunk 1 above)
Comment (dracos) [inline t/app/controller/report_new.t]: This should be covered, as the normal email test in the rest of this file is looking for a string that isn't in this email, iyswim.

**Pull request merged**